src/Custom-sample.h

```diff
@@ -98,8 +98,8 @@
                                                                             //   8 = Generic HTTP
                                                                             //   9 = FHEM HTTP
 
-#define DEFAULT_PIN_I2C_SDA                     4
-#define DEFAULT_PIN_I2C_SCL                     5
+#define DEFAULT_PIN_I2C_SDA                     13
+#define DEFAULT_PIN_I2C_SCL                     16
```

Comment:
Do not change the I2C pins in Custom-sample.h, especially not to these pins as GPIO-16 cannot be used for I2C on ESP8266.